README.md

```diff
@@ -1,126 +1,201 @@
-<a href="https://github.com/robocorp/robo/">
-  <img src="./docs/include/logo.svg" alt="Robo Logo" width="100%">
-</a>
+![Robocorp](./docs/include/robocorp-header.svg)
 
-## Robo is an all-in-one Python automation framework
+<samp>[Docs](https://robocorp.com/docs) | [Blog](https://robocorp.com/blog) | [Examples](https://robocorp.com/portal) | [ReMark](https://chat.robocorp.com) | [Courses](https://robocorp.com/docs/courses) | [Slack](https://robocorp-developers.slack.com/) | [Youtube](https://www.youtube.com/@Robocorp) | [𝕏](https://twitter.com/RobocorpInc)</samp>
 
-### Create, develop, run, and deploy your automation code with Robo - and operate it all with [Control Room](https://robocorp.com/products/control-room)
+[![PyPI - Version](https://img.shields.io/pypi/v/robocorp?label=robocorp&color=%23733CFF)](https://pypi.org/project/robocorp)
+[![PyPI - Version](https://img.shields.io/pypi/v/robocorp-action-server?label=action-server&color=%23733CFF)](https://pypi.org/project/robocorp-action-server)
+[![Downloads](https://static.pepy.tech/badge/robocorp/month)](https://pepy.tech/project/robocorp)
+[![GitHub issues](https://img.shields.io/github/issues/robocorp/robocorp?color=%232080C0)](https://github.com/robocorp/robocorp/issues)
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-<br/>
-<details open="open">
-<summary>Table of Contents</summary>
+# Create, deploy and operate 🐍 Python AI Actions <br/> and 🤖 Automations anywhere.
 
-- [What is Robo?](#what-is-robo)
-- [Libraries](#libraries)
-- [CLI](#cli)
-- [Licence](#licence)
+Robocorp is the easiest way to extend the capabilities of AI agents, assistants and copilots with custom actions, written in Python. Create and deploy tools, skills, loaders and plugins that securely connects any AI Assistant platform to your data and applications.
 
-</details>
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="./docs/include/robocorp-flow-dark.webp">
+  <img alt="Text changing depending on mode. Light: 'So light!' Dark: 'So dark!'" src="./docs/include/robocorp-flow-light.webp">
+</picture>
+
+Looking for a replacement to RPA? Head over to our [Enterprise Python Automation site](https://robocorp.com/docs/quickstart-guide) for more.
```

Comment:
FYI, @tonnitommi 

[Related to your comment on Notion](https://www.notion.so/robocorptechnologies/New-Robocorp-repo-07d02cb760a9450f8ce83e6f5e5304ab):
> Internal battle in my head where to link. I am thinking Getting started on Docs. ANy better ideas?

Reply:
It is either this or `https://robocorp.com/docs`, at this time I'd just go with the one here